code/game/machinery/computer/orders/order_computer/order_computer.dm

```diff
@@ -120,6 +120,10 @@ GLOBAL_LIST_EMPTY(order_console_products)
 				return
 			if(!purchase_items(used_id_card))
 				return
+			//So miners cant spam buy crates for a very low price
+			if(get_total_cost() < CARGO_CRATE_VALUE)
+				say("For the LTSRBT delivery order needs to cost more or equal to [CARGO_CRATE_VALUE] points!")
```

Comment:
This is also called when purchasing through the shuttle normally, so perhaps this message should be generalized?

Also the official conversion cost of mining points to credits is ``1cr = 2 mining points`` (set by ``/datum/orderable_item/consumables/space_cash`` but I don't think setting the minimum points you need to buy something as 400 would be very fun.

Reply:
> Also the official conversion cost of mining points to credits is 1cr = 2 mining points (set by /datum/orderable_item/consumables/space_cash but I don't think setting the minimum points you need to buy something as 400 would be very fun.

You are confusing things, no conversion is happening its at face value

> This is also called when purchasing through the shuttle normally, so perhaps this message should be generalized?

yeah making it more generalized

Reply:
> You are confusing things, no conversion is happening its at face value

A crate with mining items costing 200 mining points then selling the crate for 200 credits means 200 credits = 200 mining points - the cost of the items inside of it

Spending 200 mining points on crates means you're making double the cash than if you were to buy the 1000 credit bill, because you're turning 200 points to 200 credits, instead of 2000 points for 1000 credits.

sorry if this sounds delusional im running on sleep deprivation

Reply:
Then lets put this on CARGO_CRATE_VALUE * 1.5 instead?